src/app/_mermaid_common/forms/directives/validators/futureyear.dir.js

```diff
@@ -0,0 +1,31 @@
+angular.module('mermaid.forms').directive('futureyear', function() {
```

Comment:
I think we should call this `futuredate` since it's not just year that we're testing